src/ClientConfig.cc

```diff
@@ -44,6 +44,12 @@ class ignition::fuel_tools::ClientConfigPrivate
                 homePath, ".ignition", "fuel");
 
             this->servers.push_back(ServerConfig());
+
+            // Add in fuel.gazebosim.org as another default server config.
+            ServerConfig gzServerConfig;
+            gzServerConfig.SetUrl(common::URI("https://fuel.gazebosim.org"));
+            gzServerConfig.SetVersion("1.0");
+            this->servers.push_back(gzServerConfig);
```

Comment:
Just for completeness, is there an advantage to hardcoding it here as opposed to there

https://github.com/gazebosim/gz-fuel-tools/blob/b3c253c0177192e6a64683d9ccd1559de3a521e6/conf/config.yaml#L3-L6

In theory, I think that hardcoding it here makes it difficult, if not impossible for users of other servers to disable `gazebosim.org`. Not sure if any users at all care about that, but it's a use case supported to an extent.

Reply:
There are a couple problems associated with the yaml config file.

1. I don't think it's possible to use the yaml config file when running `ign gazebo`.
2. I believe the config file is only used with `ign fuel` if the `--config` option is specified.
3. If either `fuel.ignitionrobotics.org` or `fuel.gazebosim.org` are missing in the ClientConfig, then worlds/models that use both URIs will not load.

I don't see an easy way around this.